tools/wpt/run.py

```diff
@@ -493,6 +495,26 @@ def setup_wptrunner(venv, prompt=True, install_browser=False, **kwargs):
     setup_cls = product_setup[kwargs["product"]](venv, prompt, sub_product)
     setup_cls.install_requirements()
 
+    if kwargs["affected"]:
+        revish = kwargs["affected"]
+        files_changed, _ = testfiles.files_changed(revish, include_uncommitted=True, include_new=True)
+        # TODO: honor --no-manifest-update and other manifest args
+        tests_changed, tests_affected = testfiles.affected_testfiles(
+            files_changed,
+            set(["conformance-checkers", "docs", "tools"]), # TODO: dedupe
+        )
+        test_list = list(tests_changed | tests_affected)
+        if not test_list:
+            # TODO: make something show up in all loggers, and also still write wpt_report.json?
+            logger.info("no affected tests")
+            sys.exit(1)
+        test_list = [os.path.relpath(item, wpt_root) for item in test_list]
+        test_list.sort()
+        if len(kwargs["test_list"]):
```

Comment:
I was thinking it should be the union of the test list and the affected tests. I think that makes some sense for interactive use.

Reply:
Yeah, that's good, saves me the trouble of writing a good error message.